src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -52,7 +52,7 @@ internal abstract partial class AbstractFlowPass<TLocalState, TLocalFunctionStat
         /// 'MethodParameters', 'MethodThisParameter' and 'AnalyzeOutParameters(...)' should be used
         /// instead. _symbol is null during speculative binding.
         /// </summary>
-        protected readonly Symbol _symbol;
```

Comment:
>readonly Symbol _symbol [](http://example.com/codeflow?start=18&length=23)

We already have a `CurrentSymbol` field that is mutable. Do we need both fields to be mutable?

Reply:
Yes, I think so, as it drives separate APIs, like `MethodParameters` which are used by existing helpers for nullability checks.